### ISSUE TITLE
Autocomplete not finding labels in listing code blocks

### ISSUE THREAD

**Issue by fuhrmanator** [quarto-dev/quarto-vscode#101]
See #100 for the sample file.

The autocomplete `@` should find the `fig-sd` defined in the PlantUML diagram, right? I don't see it on my list of proposed labels.

**Comment by jjallaire** [comment]
No, quarto-cli doesn't currently look for figure labels on code blocks (as they aren't valid in that context). I realize that in this case you are generating a "valid" figure later on, but our crossref indexing doesn't currently anticipate this. I'll leave this open in any case as an aspiration.
